common/src/main/java/io/quarkus/ts/openshift/common/OpenShiftTestExtension.java

```diff
@@ -177,8 +177,7 @@ private void doBeforeAll(ExtensionContext context) throws Exception {
                 // using `tar` works around that nicely, because the tarball created with this command will have
                 // a single root directory named `target`, which the S2I builder image will unpack to `/deployments`
                 new Command("tar", "czf", "app.tar.gz", "target").runAndWait();
-                new Command("oc", "start-build", getAppMetadata(context).appName, "--from-archive=app.tar.gz", "--follow")
-                        .runAndWait();
+                new Command("oc", "start-build", getAppMetadata(context).appName, "--from-dir=target", "--follow").runAndWait();
```

Comment:
Does this belong to this PR?

Reply:
I think that's required for Quarkus 1.11 support. In which case, we could remove the `tar czf app.tar.gz target` command, that's no longer necessary.

Reply:
And the comment above, too :-)

Reply:
Ah it's already in #163, so it should be removed from here.